tests/unit/rss/test_views.py

```diff
@@ -64,5 +67,40 @@ def test_rss_packages(db_request):
     project3.created = datetime.date(2013, 1, 1)
     ReleaseFactory.create(project=project3)
 
-    assert rss.rss_packages(db_request) == {"newest_projects": [project3, project1]}
+    assert rss.rss_packages(db_request) == {
+        "newest_projects": tuple(zip((project3, project1), (None, None)))
+    }
     assert db_request.response.content_type == "text/xml"
+
+
+def test_format_author(db_request):
+    db_request.find_service = pretend.call_recorder(
+        lambda *args, **kwargs: pretend.stub(
+            enabled=False, csp_policy=pretend.stub(), merge=lambda _: None
+        )
+    )
+
+    db_request.session = pretend.stub()
+
+    project = ProjectFactory.create()
+    release = ReleaseFactory.create(project=project)
+
+    release.author = None
+    release.author_email = "noreply@pypi.org"
+    assert rss._format_author(release) == release.author_email
+
+    release.author = "No Reply"
+    release.author_email = "noreply@pypi.org"
+    assert rss._format_author(release) == "No Reply <noreply@pypi.org>"
+
+    release.author = "Nön Äscii"
+    release.author_email = "noreply@pypi.org"
+    assert rss._format_author(release) == "Nön Äscii <noreply@pypi.org>"
+
+    release.author = "Author One, Author Two, Author Three"
+    release.author_email = "noreply@pypi.org, noreply@pypi.org, noreply@pypi.org"
+    assert rss._format_author(release) == release.author_email
```

Comment:
Nit: this could also include the names in the form `"No Reply <noreply@pypi.org>"` as well to be consistent, unless I'm missing something?

Reply:
That would be nice, but would require opening the can of worms of heuristic parsing of author and email fields to look for multiple entries. Do we want to support semicolon as a separator in addition to comma? Do we support both mixed in one string (e.g. `foo, bar; quux` contains three or two elements -- if two, which two)? And author names can contain at least commas.

Of course we could try it out for some simple cases, e.g. split both author and author_email on commas, and if the lists are of equal length, zip-format addresses.

Reply:
Added a couple of commits along these lines.

Reply:
Having a look at the test database rows for `author` I can already see examples that would break when splitting on `,` like `Cloudistics, Inc.` and similar examples in other languages. There's also rows including the email addresses and even str'd lists of authors i.e. `['John Doe', 'Jane Doe']`.

I don't think we can reliably parse these into `(name, email)` pairs.

`author_email` seems to be more structured, althought there's quite a bit of `UNKNOWN` entries. Maybe we should just rely on that and filter the `UNKNOWN` and nulls?

Reply:
Done, rebased and simplified.